spec/controllers/v1.0/requests_controller_spec.rb

```diff
@@ -4,6 +4,10 @@
   let(:encoded_user) { encoded_user_hash }
   let(:tenant) { create(:tenant) }
 
+  let(:headers_with_admin)     { default_headers.merge(described_class::PERSONA_HEADER => described_class::PERSONA_ADMIN) }
```

Comment:
@bzwei This looks good @syncrou is trying to add regular users and admin users with a trait. Admin users would have the is_org_admin set so we might want to get the right user in there

Reply:
our admin means Approver Administrator, not org_admin, nor rbac_admin.